src/bp-templates/bp-nouveau/includes/messages/ajax.php

```diff
@@ -2680,3 +2683,129 @@ function bb_nouveau_ajax_recipient_list_for_blocks() {
 		)
 	);
 }
+
+/**
+ * Function which get moderated recepient list when click on block a member in the message screen.
+ */
```

Comment:
@jitendrabanjara1991 Add all the detail in dock block

Reply:
@chetansatasiya Added since tag and return type there